### ISSUE TITLE
Virustotal

### ISSUE THREAD

**Issue by skykiing** [ElrohirGT/VideoCompresser#2]
So I'm just a novice user, found your program looks awesome want to use it. Uploaded to virustotal and _**1 security vendor and no sandboxes flagged this file as malicious**_

1/67 screams false positive just wondering if you could give me some peace of mind since I'd really like to try out your progra
![question](https://user-images.githubusercontent.com/88942017/150738064-0979255f-f4c9-43bd-a28a-962e4b360e82.png)
m

**Comment by ElrohirGT** [comment]
Wow, I've never really used one of those web pages before. I also think that if this program was malicious you shouldn't really trust what I'm saying but still I'll try to make a case for myself. I made this program mainly because I have a really poor internet connection and I needed to upload a lot of video homework (basically every fricking class wanted a video explaining the homework on youtube), and this compresser really helped improving the upload speed but also helping me have a better archive of all the videos, although if I'm honest now that school's over I'm almost certain I'll delete them since they're already on the cloud.

In my opinion, the site has tagged you maybe because I include some ffmpeg binaries that are the ones I use to compress the video, but honestly I don't know haha as I said earlier I've never really used one of this. By the way, you remembered me, I need to make a README, so I will do that right now. Also, I've been working on a new version but the improvements are really only visible on windows because on mac the ui is broken, I'll post that later.
